src/app/_components/Header/Header.js

```diff
@@ -1,5 +1,110 @@
+'use client';
+
 import React from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import OutlinedBtn from '../OutlinedBtn';
+import * as logo from '../../../../public/assets/icons/logo.svg';
+import InstagramComponent from '../Icons/Instagram';
+import FacebookComponent from '../Icons/Facebook';
+import XComponent from '../Icons/XSocial';
 
 export default function Header() {
-  return <header>Header</header>;
+  //  Basic styles
+  const flexBetween = 'flex flex-row items-center justify-between';
+  const flexCenter = 'flex flex-row items-center justify-center';
+  const basicLink = 'no-underline list-none cursor-pointer';
+  const listItemText = 'text-primary-700 text-p2 font-bold hover:text-primary-500';
+  const iconSizes = 'w-[24px] h-[24px]';
+  const iconColors = 'text-primary-700 hover:text-primary-500';
+  const transition = 'transition duration-200 ease-in-out';
+
+  return (
+    <nav
+      className={`${flexBetween} w-full border-b-[1px] border-b-gray-300 bg-primary-100 px-20 py-4`}
+    >
+      <Link
+        href="/"
+        aria-label="Reload main page on logo click"
+        className={`${basicLink}`}
+      >
+        <Image
+          src={logo}
+          alt="Mayak logo"
+          aria-label="Mayak logo"
+          priority={true}
+          className="h-[74px] w-[129px]"
+        />
+      </Link>
+      <div className={`${flexCenter} gap-6`}>
+        <ul role="list" className={`${flexCenter} list-none gap-4`}>
+          <li className="px-3 py-1 ">
+            <Link
+              href="/specialists"
+              aria-label="Open specialists page on this link click"
+              className={`${basicLink}`}
+            >
+              <p className={`${listItemText} ${transition}`}>Спеціалісти</p>
+            </Link>
+          </li>
+          <li className="px-3 py-1">
+            <Link
+              href="/events"
+              aria-label="Open events page on this link click"
+              className={`${basicLink}`}
+            >
+              <p className={`${listItemText} ${transition}`}>Події</p>
+            </Link>
+          </li>
+        </ul>
+        <div className={`${flexBetween} gap-6`}>
```

Comment:
@Yivgha I saw similar links in the footer, so I suggest you extract this markup with social links into a separate component, like `SocialLinks`.

Reply:
done